src/shared/components/challenge-listing/Filters/ChallengeFilters.jsx

```diff
@@ -49,6 +50,7 @@ export default function ChallengeFilters({
   if (filterState.events && filterState.events.length) filterRulesCount += 1;
   if (filterState.tags && filterState.tags.length) filterRulesCount += 1;
   if (filterState.types && filterState.types.length) filterRulesCount += 1;
+  if (activeBucket === BUCKETS.ALL && filterState.status && filterState.status !== 'Active') filterRulesCount += 1;
```

Comment:
Previous issue back again: https://github.com/topcoder-platform/community-app/issues/5083#issuecomment-709555852

Please test locally to make sure all working well before mark as ready.

Thanks.

Reply:
sorry i tested a different string and pasted here different string. now its corrected